src/main/webapp/app/course/learning-paths/learning-path-management/learning-path-health-status-warning.component.html

```diff
@@ -0,0 +1,9 @@
+<div *ngIf="status" class="card text-bg-warning mb-3">
+    <div class="card-body">
+        <h5 class="card-title">{{ getWarningTitle(status) | artemisTranslate }}</h5>
+        <p class="card-text">{{ getWarningBody(status) | artemisTranslate }}</p>
+        <button *jhiHasAnyAuthority="['ROLE_INSTRUCTOR', 'ROLE_ADMIN']" type="button" class="btn btn-primary" (click)="onButtonClicked.emit()">
```

Comment:
Is the component visible for other user groups, e.g., TAs?

Reply:
no